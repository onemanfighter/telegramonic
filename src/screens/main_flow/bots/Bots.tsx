const Bots = () => {
  return <div>Bots</div>;
};

export default Bots;
