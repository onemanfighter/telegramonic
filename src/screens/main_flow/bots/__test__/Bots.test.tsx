import { render } from '@testing-library/react';
import Bots from '../Bots';

describe('Bots screen', () => {
  it('should render correctly', () => {
    const { container } = render(<Bots />);

    expect(container).toMatchSnapshot();
  });
});
