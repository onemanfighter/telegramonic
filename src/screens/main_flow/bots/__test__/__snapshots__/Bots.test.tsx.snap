// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bots screen should render correctly 1`] = `
<div>
  <div>
    Bots
  </div>
</div>
`;
